.github/workflows/ci.yaml

```diff
@@ -0,0 +1,70 @@
+#
+# Copied verbatim from Tide and then slightly modified. Great work everyone!
```

Comment:
This isn't the slightest bit sarcastic. I know where to look when I need inspiration for developer tooling. Typically it's any project that @wycats is involved in. :heart: